core/client/fs/src/test/java/alluxio/client/file/AlluxioFileInStreamTest.java

```diff
@@ -775,6 +788,31 @@ public BlockInStream answer(InvocationOnMock invocation) throws Throwable {
     }
   }
 
+  @Test
+  public void getPos() throws Exception {
+    assertEquals(0, mTestStream.getPos());
+    mTestStream.read();
+    assertEquals(1, mTestStream.getPos());
+    mTestStream.read(new byte[(int) mFileSize], 0, (int) mFileSize);
+    assertEquals(mFileSize, mTestStream.getPos());
+    mTestStream.close();
```

Comment:
If we put the `mTestStream.close();` to the `@After` method, it can be sure to executed exactly.

Reply:
fixed